.github/workflows/bazel.yml

```diff
@@ -0,0 +1,29 @@
+name: Bazel
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: ['*']
+
+jobs:
+  MacOS:
+    strategy:
+      matrix:
+        xcode_version: ['12.5.1', '13.0', '13.2.1']
+    runs-on: macos-11
+    env:
+      DEVELOPER_DIR: /Applications/Xcode_${{ matrix.xcode_version }}.app
+    steps:
+      - uses: actions/checkout@v2
+      - run: bazelisk test //Tests/SourceKittenFrameworkTests:UnitTests
+  Linux:
+    strategy:
+      matrix:
+        tag: ['5.3', '5.4', '5.5']
```

Comment:
Same here, no need for a matrix, just validating on the latest stable Swift version should be fine.